scripts/evaluate_chunk_size.py

```diff
@@ -0,0 +1,148 @@
+import os
+import uuid
+import time
+
+import chromadb
+from openai import OpenAI
+
+
+EMBEDDING_COLLECTION_NAME = "learn"
+chunk_sizes = ["paragraph", "half_a_paragraph", "quarter_a_paragraph", "sentence"]
+
+chromadb_client = chromadb.HttpClient(
+    host=os.getenv("CHROMADB_HOST", "chromadb"),
+    port=8000,
+    ssl=False,
+    headers=None,
+    settings=chromadb.config.Settings(),
+    tenant=chromadb.config.DEFAULT_TENANT,
+    database=chromadb.config.DEFAULT_DATABASE,
+)
+
+openai_client = OpenAI(api_key=os.getenv("OPENAI_API_KEY"))
+chromadb_client = chromadb.HttpClient(
+    host=os.getenv("CHROMADB_HOST", "chromadb"),
+    port=8000,
+    ssl=False,
+    headers=None,
+    settings=chromadb.config.Settings(),
+    tenant=chromadb.config.DEFAULT_TENANT,
+    database=chromadb.config.DEFAULT_DATABASE,
+)
+
+# Questions to ask about the text
+f_query = open("../data/RAG_chunk_evaluation_questions.txt", "r")
+my_queries = f_query.read().split("\n")
+f_query.close()
+
+unique_file_id = str(int(time.time()))
```

Comment:
I think renaming `unique_file_id` -> `timestamp` would make it clear what it is.

Reply:
when I see line 49, I have no idea what `unique_file_id` is. Naming it `timestamp` might remove the confusion, and looking at line 49 is sufficient, i.e., I don't have to look for `unique_file_id`'s definition in this source code.